tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -322,6 +327,10 @@ test = Matrix(test_cut[:, remove_names]);
 
     # Calculate all the mu terms.
     mu = intercept .+ x * coefficients
+    # for i in 1:n_obs
+    #     y[i] ~ Normal(mu[i], sqrt(σ²))
+    # end
+    # vectorized implementation
```

Comment:
IMO it's not good to add commented code. It will not be run and hence is not tested. Moreover, it increases the code complexity. If one wants to show an alternative, a separate model, sampling etc. would be better.

Reply:
```suggestion
```